backend/src/scripts/db-scripts.js

```diff
@@ -0,0 +1,106 @@
+const mongoose = require('mongoose');
+const Course = require('../models/course');
+
+
+// duplicate course documents with new ObjectId 
+const duplicateCourses = async (courses, numDup) => {
+    for (let course of courses) {
+        console.log(`duplicating ${course.dept} ${course.num}: ${course.title}...`);
+        for (let i = 0; i < numDup; i++) {
+            course._id = mongoose.Types.ObjectId();
+            course.isNew = true;
+            await course.save()
+        }
+    }
+}
+
+const removePrevQuartersField = async (courses) => {
+    for (let course of courses) {
+        console.log(`removing prevQuarters in ${course.dept} ${course.num}: ${course.title}`);
+        course.prevQuarters = undefined;
+        await course.save();
+    }
+}
+
+
+
+// RUN WHEN COURSES COLLECTION IS INITIALIZED
+const duplicateNecessaryCourses = async () => {
+    let specialTopicCourses = await Course.find({
```

Comment:
These are not changed, so they are const, not let.

Reply:
@anthopark

Reply:
gotcha I'll make this change